### PR TITLE
New doc: HTTP: Setup for server communication

### PR DESCRIPTION
**Pull request by trekladyone**
New document extracted from the original **Communicating with backend services using HTTP** document, which is to be retired.

### CODE REVIEW

File: aio/content/guide/http-setup-server-communication.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # HTTP: Setup for server communication
 
-Before you can use `HttpClient`, you need to import the Angular `HttpClientModule`.
-Most apps do so in the root `AppModule`.
+Needs an introductory paragraph.
+
+## Setup server communication
+
+Before `HttpClient` can be used, you need to import the Angular `HttpClientModule`. Most apps do so in the root `AppModule`.
```
Comment (eneajaho) [inline aio/content/guide/http-setup-server-communication.md:7]: Aren't the new docs supposed to include the new apis? like: provideHttpClient() ?

Comment (trekladyone) [reply aio/content/guide/http-setup-server-communication.md:7]: Within the scope of this project (Project Gummybears), we are breaking down larger documents to make the content more modular.  We will not be editing the content at this stage.

**Comment by atscott** [comment]
If this is supposed to be part of a larger refactor, I would suggest a single PR for the code moves. You can have separate commits, but I don't think it's going to make a lot of sense to create separate PRs for each section where all you're doing is copy/pasting the document to a new file.

**Review by atscott — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 4adc790a81ed41f16c37754f73a99c0e7ae80beb.

**Pull request closed without merging**